src/main/webapp/search-script.js

```diff
@@ -0,0 +1,122 @@
+function collectUnitsFromSearch() {
+  const inputElems = document.getElementsByName("search-units");
+  const addedUnits = []
```

Comment:
Nit: add a semi-colon

Seems that prettier didn't get run somehow? Should've done that for you

Reply:
It seems like prettier just isn't formatting my .js file, I'll go through and manually format it